aidb/engine/approx_select_engine.py

```diff
@@ -76,22 +76,25 @@ async def get_inference_results(self, query:Query, sampled_index: List[int], con
         query_no_where
     )
 
-    all_df = await conn.run_sync(lambda conn: pd.read_sql(text(all_query_add_filter_key.sql_str), conn))
-    # drop duplicated columns, this will happen when 'select *'
-    all_df = all_df.loc[:, ~all_df.columns.duplicated()]
-    all_df.set_index(VECTOR_ID_COLUMN, inplace=True, drop=True)
+    async with self._sql_engine.begin() as conn:
```

Comment:
Why is this changed?

Reply:
Similar to the issue encountered with approx_aggregation in MySQL database. I will separate them out.